component/mcsdadmin/templates/organization_list.html

```diff
@@ -1,27 +1,34 @@
 {{define "main"}}
-<div class="action-bar">
-    <div>
-        <a href="/mcsdadmin/organization/new">New Organization</a>
-    </div>
+<div class="d-flex justify-content-between align-items-center mb-3">
+    <h2>Organizations</h2>
+    <a href="/mcsdadmin/organization/new" class="btn btn-primary">New Organization</a>
 </div>
-<div class="main-view">
-    <table>
-        <thead>
-        <tr>
-            <th scope="col">Name</th>
-            <th scope="col">Type</th>
-            <th scope="col">Active</th>
-        </tr>
-        </thead>
-        <tbody>
-        {{range .Organizations}}
-        <tr>
-            <th scope="row">{{ .Name }}</th>
-            <td>{{.Type}}</td>
-            <td>{{ .Active }}</td>
-        </tr>
-        {{end}}
-        </tbody>
-    </table>
+<div class="card">
+    <div class="card-body">
+        <table class="table table-striped table-hover">
+            <thead class="thead-light">
+            <tr>
+                <th scope="col">Name</th>
+                <th scope="col">Type</th>
+                <th scope="col">Active</th>
+            </tr>
+            </thead>
+            <tbody>
+            {{range .Organizations}}
+            <tr>
+                <th scope="row">{{ .Name }}</th>
+                <td>{{.Type}}</td>
+                <td>
+                    {{if .Active}}
```

Comment:
This is a bug I think, .Active is a string it's zero value is empty string so the status here will always show the active badge. I'll implement a fix.